locations/kranold18.yml

```diff
@@ -0,0 +1,100 @@
+---
+location: kranold18
+location_nice: Kranoldstr. 18
+latitude: 52.513189
+longitude: 13.464245
+community: true
+
+hosts:
+
+  - hostname: kranold18-core
+    role: corerouter
+    model: 'avm_fritzbox-4040'
+    wireless_profile: kranold18
+
+  - hostname: kranold18-ap1
+    role: ap
+    model: 'ubnt_nanostation_m5_xm'
+    wireless_profile: mesh_only
+
+ipv6_prefix: "2001:bf7:820:2800::/56"
+
+networks:
+  - vid: 20
+    role: mesh
+    name: mesh_philmel
+    prefix: 10.31.235.128/29
+    ipv6_subprefix: -3
+    mesh_metric: 2048
```

Comment:
making the only link worse than default does not make sense. please just remove this line